blog/fb4/index.qmd

```diff
@@ -0,0 +1,162 @@
+
+```python
+Module(
+    body=[
+        FunctionDef(
+            name='triangle',
+            args=arguments(
+                posonlyargs=[],
+                args=[
+                    arg(arg='a'),
+                    arg(arg='b'),
+                    arg(arg='c')],
+                kwonlyargs=[],
+                kw_defaults=[],
+                defaults=[]),
+            body=[
+                If(
+                    test=Compare(
+                        left=Name(id='a', ctx=Load()),
+                        ops=[
+                            Eq()],
+                        comparators=[
+                            Name(id='b', ctx=Load())]),
+                    body=[
+                        If(
+                            test=Compare(
+                                left=Name(id='b', ctx=Load()),
+                                ops=[
+                                    Eq()],
+                                comparators=[
+                                    Name(id='c', ctx=Load())]),
+                            body=[
+                                Return(
+                                    value=Constant(value='Equilateral'))],
+                            orelse=[
+                                Return(
+                                    value=Constant(value='Isosceles'))])],
+                    orelse=[
+                        If(
+                            test=Compare(
+                                left=Name(id='b', ctx=Load()),
+                                ops=[
+                                    Eq()],
+                                comparators=[
+                                    Name(id='c', ctx=Load())]),
+                            body=[
+                                Return(
+                                    value=Constant(value='Isosceles'))],
+                            orelse=[
+                                If(
+                                    test=Compare(
+                                        left=Name(id='a', ctx=Load()),
+                                        ops=[
+                                            Eq()],
+                                        comparators=[
+                                            Name(id='c', ctx=Load())]),
+                                    body=[
+                                        Return(
+                                            value=Constant(value='Isosceles'))],
+                                    orelse=[
+                                        Return(
+                                            value=Constant(value='Scalene'))])])])],
+            decorator_list=[])],
+    type_ignores=[])
```

Comment:
I cut the example of the AST because it isn't very readable to the audience, plus I was struggling on how exactly to incorporate it within my blogpost. I decided to cut it entirely as I also believed it directed focus away from the overall messages and themes of the reading.